src/IfSharp.Kernel/helpers/paket.dependencies

```diff
@@ -0,0 +1,4 @@
+source https://www.nuget.org/api/v2
+
+nuget FSharp.Data.TypeProviders 
+nuget XPlot.Plotly 
```

Comment:
@dsyme I'm open including more. I checked and FsLab comes down as about 100MB.

There's a bit of question of how to let the user change versions. At the moment we've been saying they need to go edit this paket.dependencies file. Perhaps if we add helper functions to let users alter the dependencies inside the notebook? Then Paket will make changes as needed.